react/typings/react.d.ts

```diff
@@ -0,0 +1,7 @@
+import 'react'
+
+declare module 'react' {
+  interface ImgHTMLAttributes<T> extends HTMLAttributes<T> {
+    loading?: 'auto' | 'eager' | 'lazy'
```

Comment:
React types does not have this attribute. This extend the type definitions.

Reply:
Noob question here: how safe is this when taking browser compatibility into consideration? Old browsers would just ignore this attribute and work normally?

Reply:
@wdsrocha that's right

That's a helpful article on the subject:
https://web.dev/native-lazy-loading/#browser-compatibility